templates/kafka.default.erb

```diff
@@ -17,3 +17,6 @@ JMX_PORT=<%= @jmx_port %>
 <% if @classpath -%>
 CLASSPATH="<%= @classpath %>"
 <% end -%>
+<% if @kafka_log_directory %>
+LOG_DIR=<%= @kafka_log_directory %>
```

Comment:
This is used by kafka-server-start script, which will override the value defined in property file. If LOG_DIR unset, this value will always be /var/log/kafka.
The other option is not use this variable in property file at all.